### PR TITLE
Remove pylab references

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

This PR fixes the remaining easy cases. There are still some references in the API/docs, but they are harder to decide what to do. I think this can go in as is as an improvement.

### CODE REVIEW

File: lib/matplotlib/backends/backend_template.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
 
   import matplotlib
   matplotlib.use('xxx')
-  from pylab import *
-  plot([1,2,3])
+  import matplotlib.pyplot as plt
+  plt.plot([1,2,3])
   show()
```
Comment (QuLogic) [inline lib/matplotlib/backends/backend_template.py]: This should be `plt.show()`

File: lib/matplotlib/widgets.py [hunk 2]
```diff
@@ -1332,18 +1332,13 @@ class MultiCursor(Widget):
     Example usage::
 
         from matplotlib.widgets import MultiCursor
-        from pylab import figure, show, np
```
Comment (QuLogic) [inline lib/matplotlib/widgets.py]: The bare `show` below needs to be updated.

File: lib/matplotlib/backends/backend_template.py (hunk 1 above)
Comment (timhoffm) [reply lib/matplotlib/backends/backend_template.py]: Handled in #11481

File: lib/matplotlib/widgets.py (hunk 2 above)
Comment (timhoffm) [reply lib/matplotlib/widgets.py]: Handled in #11481

**Pull request merged**